README.md

```diff
@@ -1,24 +1,175 @@
-# README
+# Gaming_console_reservation_back_end
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<a name="readme-top"></a>
 
-Things you may want to cover:
+# 📗 Table of Contents
+- [:book: About the Project](#about-project)
+  - [:hammer_and_wrench: Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [:computer: Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [:busts_in_silhouette: Authors](#authors)
+- [:telescope: Future Features](#future-features)
+- [:handshake: Contributing](#contributing)
+- [:star:️ Show your support](#support)
+- [:pray: Acknowledgements](#acknowledgements)
+- [:question: FAQ](#faq)
+- [:memo: License](#license)
 
-* Ruby version
+# 📖 [Gaming_console_reservation_back_end] <a name="about-project"></a>
 
-* System dependencies
+**[Gaming_console_reservation_back_end]** is a backend rails api that provides service to the front-end react app to fetch data for gaming console reservation.
 
-* Configuration
+## [Link to the front-end repo](https://github.com/stephenkati/gaming_console_reservation_front_end.git)
 
-* Database creation
 
-* Database initialization
+## 🛠 Built With <a name="built-with"></a>
 
-* How to run the test suite
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Ruby on Rails</summary>
+  <ul>
+    <li><a>https://rubyonrails.org/</a></li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+### Key Features <a name="key-features"></a>
 
-* Deployment instructions
+- **[Render gaming-consoles]**
+- **[Add gaming-consoles]**
+- **[Delete gaming-consoles]**
+- **[Reserve gaming-consoles]**
 
-* ...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- Install Ruby
+- Install Rails
+- Install Postgresql
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd gaming_console_reservation_back_end
+  git clone https://github.com/stephenkati/gaming_console_reservation_back_end.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd gaming_console_reservation_back_end
+  bundle install
+```
+
+```sh
+  gem install --no-document rubocop -v '>= 1.0, < 2.0' 
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec ./spec/integration
+  rspec ./spec/requests
+  rspec ./spec/model
```

Comment:
- [ ] This command is incorrect because the correct folder is `models` not `model`:

`model`

![image](https://github.com/stephenkati/gaming_console_reservation_back_end/assets/57421823/43f8dd6f-7ade-4fec-ba33-005658896b23)

`models`

![image](https://github.com/stephenkati/gaming_console_reservation_back_end/assets/57421823/6f90e683-595b-4f01-ba99-30237a43e22c)

Kindly, correct the command so your users are able to run your models tests.  👍